app/models/manageiq/providers/azure/cloud_manager/refresh_parser.rb

```diff
@@ -1,3 +1,5 @@
+require 'azure-armrest'
+
```

Comment:
I've learned the hard way that we shouldn't do this. It causes a load ordering problem that results in the Rails cache getting cleared because of the cache_method gem:

https://github.com/ManageIQ/manageiq-providers-azure/pull/38